package.json

```diff
@@ -10,7 +10,6 @@
         "bower": "~1"
     },
     "scripts": {
-        "postinstall" : "./node_modules/.bin/bower install",
```

Comment:
to resolve our dependencies without run 2 commands https://github.com/ngutils/cr-acl/blob/master/bower.json